packages/runtime/src/Runtime.ts

```diff
@@ -148,6 +151,7 @@ export abstract class Runtime<TConfig extends RuntimeConfig = RuntimeConfig> {
         this.eventBus.publish(new RuntimeInitializedEvent());
     }
 
+    // TODO: well this seems redundant
     protected initInfrastructure(): void | Promise<void> {
         return;
     }
```

Comment:
@jkoenig134 Can these no-op functions be removed?

Reply:
nope. these are protected and used in the connector.

Reply:
*overwritten